cypress/support/fragments/inventory/inventorySearch.js

```diff
@@ -1,18 +1,52 @@
-import { MultiColumnList, MultiColumnListCell, MultiSelect, Pane } from '../../../../interactors';
+import { MultiColumnList, MultiColumnListCell, Pane, Accordion, Checkbox, TextField } from '../../../../interactors';
+import InventoryActions from './inventoryActions';
 
 
-const effectiveLocationInput = MultiSelect({ id: 'multiselect-6' });
+const effectiveLocationInput = Accordion({ id: 'effectiveLocation' });
+const languageInput = Accordion({ id: 'language' });
+const keywordInput = TextField({ id: 'input-inventory-search' });
 
 export default {
```

Comment:
could you move link to this file into our wiki as best practices? For example, to this section https://wiki.folio.org/display/DQA/Interactors#Interactors-Examples